requirements.txt

```diff
@@ -20,3 +20,11 @@ textstat
 
 torch
 transformers
+TextBlob
```

Comment:
TextBlob -> textblob

Reply:
i removed it. you are right! thanks

Reply:
TextBlob : the class for text
textblob : library / mod containg the TextBlob class and nlp tools